x-pack/plugins/security_solution/public/detections/containers/detection_engine/rules/use_rule_from_timeline.tsx

```diff
@@ -206,12 +204,19 @@ export const useRuleFromTimeline = (setRuleQuery: SetRuleQuery): RuleFromTimelin
   const [urlStateInitialized, setUrlStateInitialized] = useState(false);
 
   useEffect(() => {
-    if (timelineIdFromUrl != null && !urlStateInitialized) {
+    const id =
+      timelineIdFromUrl != null
```

Comment:
Nit: Same thing as the earlier comment. Nested ternaries will get worse as rules are added

Reply:
https://github.com/elastic/kibana/pull/151091/commits/9b1523f5291920cc68d313e8a3159b195812161e